02-FirstApplicationEnvironment/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y golang go git
+RUN git clone https://github.com/TheAnig/golang_fizzbuzz
```

Comment:
This works and is the pattern I followed, but it is worth noting some servers which may be building and running your dockers do not neccessarily have access to git or you may want to build/test code which is not on the remote repo.  For this you can use COPY and the Dockerfile would typically be in the application itself.  As noted, your pattern here is fine, I provide this for information on patterns going forward.